src/Components/Search/SearchBar.tsx

```diff
@@ -85,29 +76,36 @@ const Form = styled.form`
 `
 
 // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-@track(null, {
+const SearchBar = track(null, {
   dispatch: data => Events.postEvent(data),
-})
-export class SearchBar extends Component<Props, State> {
-  public input: HTMLInputElement
+})(({ router, relay, viewer }) => {
+  const [, setEntityID] = useState(null)
+  const [, setEntityType] = useState(null)
```

Comment:
After converting to a functional component, I noticed that we are not using this state anywhere.